src/Pages/Profile/admin/EditorForm.js

```diff
@@ -67,7 +67,31 @@ export default function EditForm (props) {
                 style={{ marginTop: '5px' }}
               >
                 Reset Pages!
-              </Button>
+              </Button> */}
+              <FormGroup check inline style={{ marginTop: '5px' }}>
+                <Label check style={{ marginRight: '5px' }} id='resetPages2'>
+                  Reset Pages!
+                </Label>
+                <Input
+                  type='checkbox'
+                  id='resetPages'
+                  onClick={() => {
```

Comment:
move the on click function to it's own function above the react code.

Reply:
```js
example:
function yourFunction () {...}

<input ... onClick = {()=>{yourFunction()}} />
```